spec/system/docs_index_spec.rb

```diff
@@ -29,6 +34,7 @@
 
       it "does not show the topic excerpts by default" do
         visit("/docs")
+        expect(page).to have_css(".topic-list-item", count: 2)
         expect(page).to have_no_css(".topic-excerpt")
```

Comment:
This was passing previously because the assertion would run before the page is actually loaded.

If the page loaded very quickly for some reason, then the assertion would fail because of the interaction with the tooltips plugin.

Adding a `have_css` assertion beforehand ensures that rendering has completed before running the `have_no_css` check